pkg/config/config.go

```diff
@@ -388,6 +388,13 @@ func initConfig(config Config) {
 	config.BindEnvAndSetDefault("bosh_id", "")
 	config.BindEnvAndSetDefault("cf_os_hostname_aliasing", false)
 
+	// Cloud Foundry BBS
+	config.BindEnvAndSetDefault("cloud_foundry_bbs.url", "https://bbs.service.cf.internal:8889")
+	config.BindEnvAndSetDefault("cloud_foundry_bbs.poll_interval", 15)
```

Comment:
Should we make this a `int64(15)` like for `jmx_restart_interval `?

Reply:
We don't expect this to be so huge that we'd need int64 (in fact it would be pretty much useless with anything higher than couple thousand) - this is more similar to e.g. `kubelet_listener_polling_interval`, which isn't int64 either, so I'd prefer leaving it as it is right now.